server/src/test/java/org/elasticsearch/action/admin/indices/TransportAnalyzeActionTests.java

```diff
@@ -249,6 +249,38 @@ public void testFillsAttributes() throws IOException {
         assertEquals("<ALPHANUM>", tokens.get(3).getType());
     }
 
+    public void testMultiPhraseRequest() throws IOException {
+        AnalyzeAction.Request request = new AnalyzeAction.Request();
+        request.analyzer("standard");
+        request.text("first thing", "second try");
+        AnalyzeAction.Response analyze = TransportAnalyzeAction.analyze(request, registry, null, maxTokenCount);
```

Comment:
I think that instead of hardcoding the expected response when an index is not provided, i would compare the output when an index is provided to that of when it is not. That seems more future proof?

Reply:
That sounds good. I'll update the test using this approach.

Reply:
I updated the test based on the pr comments